Correct specification of NCEP grids 172 and 220

GeorgeGayno-NOAA opened the pull request:
Update w3fi71.f, which specifies the GRIB1 grid description section for NCEP
pre-defined grids, to correct errors with grids 172 and 220.

New unit test for w3fi71.f.

Fixes #157.

GeorgeGayno-NOAA committed:
Update the corner point latitude and orientation angle
to the correct values - grids 172 and 220.

Fixes #157.

GeorgeGayno-NOAA committed:
Add unit test stub for routine w3fi71.

Fixes #157.

GeorgeGayno-NOAA committed:
Update unit test.

Fixes #157.

GeorgeGayno-NOAA committed:
More unit tests updates for grid 172

Fixes #157.

GeorgeGayno-NOAA committed:
Add test of grid 220.

Fixes #157.

GeorgeGayno-NOAA committed:
Add some comments to unit test.

Fixes #157.

GeorgeGayno-NOAA commented:
I added a unit test. But a more robust check would include a mapping of data to grids 172 and 220 using copygb. That could be done here: https://github.com/NOAA-EMC/NCEPLIBS-grib_util/blob/develop/tests/run_copygb_tests.sh

Thoughts? The change to w3emc would need to be coordinated with the change to grib_util.

edwardhartnett commented on tests/test_w3fi71.f90:
```diff
@@ -0,0 +1,59 @@
+ program test_w3fi71
```
Awesome, thanks for the test!!

edwardhartnett commented:
Yes by all means add the test in grib_utils as well.

We can coordinate releases. We have just updated a lot of build stuff having to do with shared vs. static, and we can do a bunch of releases to get the new build stuff out everywhere.

edwardhartnett commented:
Are you ready for this to me merged? If so, click "Ready for review" and I will merge it.

GeorgeGayno-NOAA commented:
A test version of copygb was compiled on Cactus using my bug fix branch:
`/lfs/h2/emc/global/noscrub/George.Gayno/mallory/copygb/NCEPLIBS-grib_util/exec/copygb.4.mallory`

@malloryprow just tested it and said it is working.

GeorgeGayno-NOAA commented:
> Yes by all means add the test in grib_utils as well.

I added a test to copygb. For details, see: https://github.com/NOAA-EMC/NCEPLIBS-grib_util/pull/203
> 
> We can coordinate releases. We have just updated a lot of build stuff having to do with shared vs. static, and we can do a bunch of releases to get the new build stuff out everywhere.
